test/composables/useTheme.test.ts

```
@@ -1,75 +1,74 @@
   it('sets and gets the showBaseURL value', () => {
-    theme.setShowBaseURL(true)
-    const result = theme.getShowBaseURL()
+    themeConfig.setShowBaseURL(true)
+    const result = themeConfig.getShowBaseURL()
     expect(result).toBe(true)
   })
 
   it('returns the default jsonViewer deep value', () => {
-    const result = theme.getJsonViewerDeep()
+    const result = themeConfig.getJsonViewerDeep()
     expect(result).toBe(Infinity)
```

Biome

[error] Use Number.POSITIVE_INFINITY instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.POSITIVE_INFINITY instead.


(lint/style/useNumberNamespace)